### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by ericpoirier** [EclipseFdn/ecdtools.eclipse.org#152]
I'm creating this issue to track the feedback from @croundy, @brianking and @chrisguindon.

Here are my notes based on the meeting we had today.

**Global**
- [x] Use coloured logos everywhere (Eric)
- [x] Make links more prominent (Eric)
- [x] The Cloud ToolTime webinar series banner should be changed as follows: (Eric)

> Remove date: December 1, 2020
> 
> Change text to read: Join us for this fun, educational and interactive webinar series, featuring a variety of cloud development topics from community members!

**Home page**
- [x] Testimonials: Update the picture of the user (Eric)
- [x] Show Brian how to update webinar series yml file (Eric)

**Projects**
- [x] Race conditions with match-height (Chris & Eric)
- [x] Update latest release and project status (Eric)
- [x] Project page: add > (Eric)
- [x] Contact project leads to ask them to update the PMI project pages (Brian)

**Open VSX**
- [x] Committers
  - [x] Add spacing to top of Committers title (Eric)
  - [x] Truncate bio (Eric)
  - [x] Add "Read More" button that links to their account (Eric)
  - [x] Ask committers to update their profiles (Brian & Clark)
- [x] Remove Contribution Activity and Who is involved (Eric)
  - [x] Open bug to find out how we want to implement this #154  (Eric)
  - [x] Take a screenshot of the section (Eric)

**How to be listed as an adopter:**
- [x] Update the url: get-listed (Eric)
- [x] Provide a redesign (Brian & Clark)

**Members**
- [x] Remove drop down menu (Eric)

**Videos**
- [x] The menu item should be named: Videos (Eric)
- [x] All the current items are placeholders. We should get items from the CloudDev tools playlist
  - [x] Will need video links, titles, descriptions (Brian & Clark)
  - [x] Learn more: Add link of the full playlist (Eric)

**Community**

- [x] Remove Learn about membership button and put it at the top of the page (Eric)

**Comment by croundy** [comment]
Global change: 

The Cloud ToolTime webinar series banner should be changed as follows:

Remove date: December 1, 2020

Change text to read: Join us for this fun, educational and interactive webinar series, featuring a variety of cloud development topics from community members!

**Comment by croundy** [comment]
Home Page:

1. "What's New" section should be pulling from Cloud DevTools sources instead of IoT sources
2. The section that reads " Open Source Community-driven Collaboration" should have a parallax treatment as an interesting visual element

**Comment by brianking** [comment]
@ericpoirier Regarding "Ask committers to update their profiles" (and project leads), where is this data being pulled from? I'm putting together some communications to send out around this, as well as ensuring PMI pages for projects are up-to-date.

**Comment by ericpoirier** [comment]
> @ericpoirier Regarding "Ask committers to update their profiles" (and project leads), where is this data being pulled from? I'm putting together some communications to send out around this, as well as ensuring PMI pages for projects are up-to-date.

This is being pulled from https://accounts.eclipse.org and this is where the users will have to update their information.

**Comment by ericpoirier** [comment]
I started a PR to include the changes that we gather in this issue. You can see the changes here.
https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/

And you can keep track of what have been done based on the elements that I checked on this list.

**Comment by ericpoirier** [comment]
> Home Page:
> 
> 1. "What's New" section should be pulling from Cloud DevTools sources instead of IoT sources

Can you supply links and titles you'd like to use for the videos? The list of videos located on the home page and on the Videos page is generated from a .yml file in the data folder which is manually maintained. 

> 2. The section that reads " Open Source Community-driven Collaboration" should have a parallax treatment as an interesting visual element

There is already Parallax to that block. I'm removing it for mobile so that it loads better but the effect is there for desktop.

**Comment by croundy** [comment]
> > Home Page:
> > 
> > 1. "What's New" section should be pulling from Cloud DevTools sources instead of IoT sources
> 
> Can you supply links and titles you'd like to use for the videos? The list of videos located on the home page and on the Videos page is generated from a .yml file in the data folder which is manually maintained.

This document shows titles, a brief description and links to the videos we want to feature on the video page:

https://docs.google.com/document/d/1Z_0ofugO8tDwYtCYHK0BEmVWEQ5VsIxlsGwk5ZUVSVs/edit#

@brianking please review and feel free to edit/change

**Comment by brianking** [comment]
That list looks ok to me. An alternative would be to just show the latest n videos, on chronological order, but I am also ok with the featured approach..
The Learn More button up above linking to the playlist is not obvious, so it might make sense sense to link to it somehow underneath the last video, with text like 'Go to the full playlist'.

**Comment by ericpoirier** [comment]
> That list looks ok to me. An alternative would be to just show the latest n videos, on chronological order, but I am also ok with the featured approach..
> The Learn More button up above linking to the playlist is not obvious, so it might make sense sense to link to it somehow underneath the last video, with text like 'Go to the full playlist'.

Would you like me to remove the button at the top of the page or should we keep it and add the 2nd button at the bottom of the page?

**Comment by brianking** [comment]
> Would you like me to remove the button at the top of the page or should we keep it and add the 2nd button at the bottom of the page?

You can keep it, let's see what it looks like with both.

**Comment by ericpoirier** [comment]
I updated the videos page on the staging page.

https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/resources/videos/

If there's nothing else to be updated from the videos page, we can mark this item as "done" in the list at the top of this page.

**Comment by croundy** [comment]
> I updated the videos page on the staging page.
> 
> https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/resources/videos/
> 
> If there's nothing else to be updated from the videos page, we can mark this item as "done" in the list at the top of this page.

LGTM!

**Comment by ericpoirier** [comment]
So I think all we need now are the following items:

1. Provide a redesign for the "How to be listed as an adopter" page.
    https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/adopters/get-listed/

2. Ask committers to update their profiles for the Open VSX page
    https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/projects/open-vsx/

3. Contact project leads to ask them to update the PMI project pages

**Comment by croundy** [comment]
A couple of minor things:

1. "Change text to read:" was left in the text that goes with the Cloud Tool Time banner at the bottom of the page 
2.  Home page "What's New" section. The text styling for the Event/calendar is different. Can we make that consistent? Can we also pull in the description of that event that is posted on the Crowdcast registration page? https://www.crowdcast.io/e/cloud-tool-time-sept7/register:

The presentation introduces the impulse Waveform and Event Viewer, outlines the steps to migrate from the Eclipse IDE to Theia and shows possibilities for integration into own environments (Theia/HTML).

**Comment by brianking** [comment]
> So I think all we need now are the following items:
> 
>     1. Provide a redesign for the "How to be listed as an adopter" page.
>        https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/adopters/get-listed/

This is ready here:
https://docs.google.com/document/d/1Li_bib1Ql2BwvpIgL2P2-Cqc1RCd_dNvDi6Zu4nbiD0/edit?usp=sharing
(https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/156)

>     2. Ask committers to update their profiles for the Open VSX page
>        https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/projects/open-vsx/

I'm not sure the best way to approach this. It feels like 1-1 is better so that they are more compelled to act on it, but that would be a lot of work to do. Any ideas?

>     3. Contact project leads to ask them to update the PMI project pages

See:
https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/155

**Comment by ericpoirier** [comment]
> A couple of minor things:
> 
> 1. "Change text to read:" was left in the text that goes with the Cloud Tool Time banner at the bottom of the page
> 2. Home page "What's New" section. The text styling for the Event/calendar is different. Can we make that consistent? Can we also pull in the description of that event that is posted on the Crowdcast registration page? https://www.crowdcast.io/e/cloud-tool-time-sept7/register:
> 
> The presentation introduces the impulse Waveform and Event Viewer, outlines the steps to migrate from the Eclipse IDE to Theia and shows possibilities for integration into own environments (Theia/HTML).

Done.

> > So I think all we need now are the following items:
> > ```
> > 1. Provide a redesign for the "How to be listed as an adopter" page.
> >    https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/adopters/get-listed/
> > ```
> 
> This is ready here:
> https://docs.google.com/document/d/1Li_bib1Ql2BwvpIgL2P2-Cqc1RCd_dNvDi6Zu4nbiD0/edit?usp=sharing
> (#156)

Done.

> > ```
> > 2. Ask committers to update their profiles for the Open VSX page
> >    https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/projects/open-vsx/
> > ```
> 
> I'm not sure the best way to approach this. It feels like 1-1 is better so that they are more compelled to act on it, but that would be a lot of work to do. Any ideas?

I don't think there's a perfect solution for this. As you're saying, contacting each of them one by one would be a lot of work. Maybe contacting the project lead so that this person can put pressure on the committers to update their profile would be a good way to go? As well as sending an email to everyone. In the end, it's kind of out of our control if they do it or not.

> > ```
> > 3. Contact project leads to ask them to update the PMI project pages
> > ```
> 
> See:
> #155

+1 


See the changes here: https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/

**Comment by croundy** [comment]
Looking good. On the "Showcase Your Work" page, we are missing a couple of links: 

 static/assets/images/adoptors
config/adopters.json

Also perhaps reduce the font size a bit for the "Who is Considered an Adopter?" and "How Can I Get My Logo Displayed?" text

**Comment by brianking** [comment]
Also on that page (/adopters/get-listed/), the page title shows up in the tab bar as ALL CAPS while other pages do not.

**Comment by ericpoirier** [comment]
I updated that page with the missing links.

I would suggest keeping the title size as is because they are <h2> tags and to keep things consistent throughout the site I wouldn't change it for one page only. If you're saying the <h2> is too big throughout the site, then we can make that change no problem.

Let me know if you'd like to reduce the size of the <h2> on all sub pages.

Thanks.

**Comment by ericpoirier** [comment]
> I updated that page with the missing links.
> 
> I would suggest keeping the title size as is because they are
> 
> h2 tags and to keep things consistent throughout the site I wouldn't change it for one page only. If you're saying the
> ## is too big throughout the site, then we can make that change no problem.
> Let me know if you'd like to reduce the size of the
> 
> h2 on all sub pages.
> Thanks.

**Comment by ericpoirier** [comment]
Based on the discussion we had on our call, I've made the following changes:

**Webinars section:**
Buttons are now part of the slide items
Add automatic rotation
Update the data file to be more generic

**Blogs**
Update blog items based on Brian's blogs post on blogs.eclipse.org (I got only 3)
Added the Blogs menu item under Resources

**Community**
Switch the Calendar and Virtual Events icons

**Search**
Create a new issue to create a search page for ECDtools only
Remove the search from the menu

Here is the updated site with these changes.
https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/

Let me know if there's anything else.

Thanks.

**Comment by brianking** [comment]
Thanks @ericpoirier. One thing I notice is where blog posts have their own page, there is no link to view the full post.

e.g.
https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/resources/blogs/2021-07-13-5-reasons-to-adopt-eclipse-theia/

**Comment by ericpoirier** [comment]
> Thanks @ericpoirier. One thing I notice is where blog posts have their own page, there is no link to view the full post.
> 
> e.g.
> https://deploy-preview-153--eclipsefdn-ecdtools.netlify.app/resources/blogs/2021-07-13-5-reasons-to-adopt-eclipse-theia/

I updated the links and are now going to blogs.eclipse.org

**Comment by croundy** [comment]
@ericpoirier looking great! I just noticed that the links for the Cloud Tool Time footer are not active. Please add these links:

Register Now: https://www.crowdcast.io/e/cloud-tool-time-sept7/register
Join the Conversation: Change link name to "Propose a Talk" and link it to https://docs.google.com/forms/d/e/1FAIpQLScudzvlUVfXehmz-Jkivv_BkJdljSjtZLENkT16jSxTzwzwkQ/viewform

Thanks!

**Comment by jph152** [comment]
Here is the content for slide items:

https://docs.google.com/document/d/166fmk6BYrtCuIGheWtxD8TdVaOMFysdB_gqTDyCIWEs/edit?usp=sharing

@ericpoirier should the images be added as separate attachments?

**Comment by ericpoirier** [comment]
> Here is the content for slide items:
> 
> https://docs.google.com/document/d/166fmk6BYrtCuIGheWtxD8TdVaOMFysdB_gqTDyCIWEs/edit?usp=sharing
> 
> @ericpoirier should the images be added as separate attachments?

If you can provide the images separatly, that would be great.

**Comment by jph152** [comment]
![Cloud Tool Time Cards](https://user-images.githubusercontent.com/69642121/134358079-073807d3-5e1e-41c2-9aef-923843e50171.jpg)

![TheiaConLogo-Colour](https://user-images.githubusercontent.com/69642121/134358156-4d86ddf8-74af-4efa-8c77-1c0029253fb2.png)

![image_31621888-bfdb-4673-80d9-b217b62c6d0920210921_104843 (1)](https://user-images.githubusercontent.com/69642121/134358324-bcfe6330-00a8-45b7-bfef-db447b653dca.jpg)
![A (1)](https://user-images.githubusercontent.com/69642121/134358412-025b2eaa-2078-4526-ac7e-9a0d47884b83.png)
![EclipseCon-Logo-V2-2021-Orange White transparent background (1)](https://user-images.githubusercontent.com/69642121/134358621-88cef60f-8778-422e-9a68-72126dd4e2b9.png)

**Comment by ericpoirier** [comment]
Some images are not very readable where the images are currently going. 

![image](https://user-images.githubusercontent.com/623165/134361493-cc4af2b3-6937-46ba-816a-159fa85cf996.png)

Can we have another image here?

Thanks.

**Comment by croundy** [comment]
@ericpoirier The space on the left where the "Cloud Tool Time" text seems like it could be used better. Is it possible to combine that space with the space where the image is located, so we can make that image larger? It would then basically just be 2 columns instead of 3 and have more visual pop. We could then add "Cloud Tool Time Webinars" as a headline above the body text on the right

**Comment by ericpoirier** [comment]
> @ericpoirier The space on the left where the "Cloud Tool Time" text seems like it could be used better. Is it possible to combine that space with the space where the image is located, so we can make that image larger? It would then basically just be 2 columns instead of 3 and have more visual pop. We could then add "Cloud Tool Time Webinars" as a headline above the body text on the right

Yeah thats a possibility but keep in mind that there's only one layout for all the slides so the other slides with smaller images are going to use the same layout. That being said, what you are suggesting would probably be better for the overall of the slides.

I'll make that change and push it to staging so you can have a look at all the slides.

**Comment by ericpoirier** [comment]
@croundy @brianking You can see the updated Highlights section here:
https://deploy-preview-164--eclipsefdn-ecdtools.netlify.app/

I updated the styles and the layout to make it work with Clark's idea.

Let me know if you'd like to make more changes.

**Comment by croundy** [comment]
@ericpoirier I'm liking it. WDYT?

**Comment by jph152** [comment]
![Cloud DevTools headshots](https://user-images.githubusercontent.com/69642121/134384070-325fba41-251e-4c52-8699-e19f98a6380a.png)
Here is Jonas Helming's photo for the featured projects/project lead section. Waiting on confirmation from Marc Nuri about his picture

**Comment by ericpoirier** [comment]
> @ericpoirier I'm liking it. WDYT?

Yeah, I like it too.

**Comment by ericpoirier** [comment]
> ![Cloud DevTools headshots](https://user-images.githubusercontent.com/69642121/134384070-325fba41-251e-4c52-8699-e19f98a6380a.png)
> Here is Jonas Helming's photo for the featured projects/project lead section. Waiting on confirmation from Marc Nuri about his picture

You can see the updated picture here:
https://deploy-preview-164--eclipsefdn-ecdtools.netlify.app/

**Comment by jph152** [comment]
![Cloud DevTools headshots (1)](https://user-images.githubusercontent.com/69642121/134506817-7f5d7944-deff-4552-ab52-5274414a410c.png)

Thanks @ericpoirie, here is the other picture for that section

**Comment by brianking** [comment]
> @brianking not sure I got around showing you how to update the carousel but to do so, you can edit the following data file:
> https://github.com/EclipseFdn/ecdtools.eclipse.org/blob/master/data/highlights.yml

Thanks, please add this to the content updating document if not already there.

**Comment by ericpoirier** [comment]
> > @brianking not sure I got around showing you how to update the carousel but to do so, you can edit the following data file:
> > https://github.com/EclipseFdn/ecdtools.eclipse.org/blob/master/data/highlights.yml
> 
> Thanks, please add this to the content updating document if not already there.

Done.
